src/tree/registries/RegistriesTreeItem.ts

```diff
@@ -98,7 +98,9 @@ export class RegistriesTreeItem extends AzExtParentTreeItem {
             context.telemetry.properties.cancelStep = 'learnHowToContribute';
             throw new UserCancelledError();
         } else if (provider.onlyOneAllowed && this._cachedProviders.find(c => c.id === provider.id)) {
-            throw new Error(`Only one provider with id "${provider.id}" is allowed at a time.`);
+            await ext.ui.showWarningMessage(`The "${provider.id}" registry provider is already connected.`);
```

Comment:
Can we use the `label` rather than the `id` as the latter is not necessarily intended for human consumption and therefore might be confusing?

Reply:
Yes, that's a better idea than what I had.

![image](https://user-images.githubusercontent.com/36966225/66769819-0fd19d00-ee84-11e9-8a30-6e49c7afda35.png)